src/Bridges/Constraint/geomean_to_relentr.jl

```diff
@@ -0,0 +1,92 @@
+MOI.get(bridge::GeoMeantoRelEntrBridge{T, F, G}, ::MOI.NumberOfConstraints{G, MOI.RelativeEntropyCone}) where {T, F, G} = 1
+MOI.get(bridge::GeoMeantoRelEntrBridge{T, F}, ::MOI.ListOfConstraintIndices{F, MOI.GreaterThan{T}}) where {T, F} = [bridge.ge_index]
+MOI.get(bridge::GeoMeantoRelEntrBridge{T, F, G}, ::MOI.ListOfConstraintIndices{G, MOI.RelativeEntropyCone}) where {T, F, G} = [bridge.relentr_index]
+
+# References
+function MOI.delete(model::MOI.ModelLike, bridge::GeoMeantoRelEntrBridge)
+    MOI.delete(model, bridge.relentr_index)
+    MOI.delete(model, bridge.ge_index)
+    MOI.delete(model, bridge.y)
+end
+
+# Attributes, Bridge acting as a constraint
+function MOI.get(model::MOI.ModelLike, ::MOI.ConstraintFunction, bridge::GeoMeantoRelEntrBridge{T, F, G, H}) where {T, F, G, H}
+    relentr_func = MOIU.eachscalar(MOI.get(model, MOI.ConstraintFunction(), bridge.relentr_index))
+    d = div(length(relentr_func) - 1, 2)
+    u_func = MOIU.remove_variable(MOIU.operate(-, T, relentr_func[end], MOI.SingleVariable(bridge.y)), bridge.y)
+    w_func = relentr_func[2:(1 + d)]
+    return MOIU.convert_approx(H, MOIU.operate(vcat, T, u_func, w_func))
+end
+MOI.get(model::MOI.ModelLike, ::MOI.ConstraintSet, bridge::GeoMeantoRelEntrBridge) = MOI.GeometricMeanCone(1 + div(MOI.dimension(MOI.get(model, MOI.ConstraintSet(), bridge.relentr_index)) - 1, 2))
+MOI.supports(::MOI.ModelLike, ::Union{MOI.ConstraintPrimalStart, MOI.ConstraintDualStart}, ::Type{<:GeoMeantoRelEntrBridge}) = true
+function MOI.get(model::MOI.ModelLike, attr::Union{MOI.ConstraintPrimal, MOI.ConstraintPrimalStart}, bridge::GeoMeantoRelEntrBridge)
+    relentr_primal = MOI.get(model, attr, bridge.relentr_index)
+    d = div(length(relentr_primal) - 1, 2)
+    y_val = MOI.get(model, attr, bridge.ge_index)
+    u_val = sum(relentr_primal[(2 + d):end]) / d - y_val
+    return vcat(u_val, relentr_primal[2:(1 + d)])
+end
+function MOI.set(model::MOI.ModelLike, attr::MOI.ConstraintPrimalStart, bridge::GeoMeantoRelEntrBridge, value)
+    u_val = value[1]
+    u_pos = -min(zero(u_val), u_val)
+    d = length(value) - 1
+    MOI.set(model, MOI.VariablePrimalStart(), bridge.y, u_pos)
+    MOI.set(model, attr, bridge.ge_index, u_pos)
+    MOI.set(model, attr, bridge.relentr_index, vcat(0, value[2:end], fill(u_pos + u_val, d)))
+    return
+end
+# Given a is dual on y >= 0 and (b, c, d) is dual on RelativeEntropyCone constraint,
+# dual on (u, w) in GeometricMeanCone is (-a, c)
+function MOI.get(model::MOI.ModelLike, attr::Union{MOI.ConstraintDual, MOI.ConstraintDualStart}, bridge::GeoMeantoRelEntrBridge)
+    u_dual = -MOI.get(model, attr, bridge.ge_index)[1]
+    relentr_dual = MOI.get(model, attr, bridge.relentr_index)
+    d = div(length(relentr_dual) - 1, 2)
+    w_dual = relentr_dual[2:(d + 1)]
+    return vcat(u_dual, w_dual)
+end
+# Given constraint dual start of (u, w), constraint dual on RelativeEntropyCone
+# constraint is (1/d, -u/d * e, -w) and dual on y >= 0 constraint is -u.
```

Comment:
From above, we have u = -a = e'd and w = c
So c = w, a = -u and  e'd = u.
To deduce that b = 1/d and d = u/n * e, we would need to know what is the dual of the RelativeEntropyCone.
Here `(1/d, -u/d * e, -w)` seems incorrect, `(1/d, w, u/d * e)` might work though. The test should have failed with the current implementation

Reply:
So you need to find b, c, d. You know that c and you know sum(d) and you know that (b, c_i, d_i) in DualExp for each i.
You can decouple the problem. First, you find b such that
-b sum exp(c_i/b) <= exp(1) * sum d_i
which can be obtained by minimizing -b sum exp(c_i/b)
Then you split sum d_i according to the weight -b exp(c_i/b) so you give `d_i = -b exp(c_i/b) / (-b sum exp(c_i/b)) * sum d_i`

Reply:
The univariate minimization should be easy since `sum -b exp(c_i/b)` is convex as it is the sum of convex functions.

Reply:
are you proposing to use say Optim.jl to solve this univariate convex minimization problem?

Reply:
For now I've updated it with your first suggestion of `(1/d, w, u/d * e)`, and added a TODO

Reply:
> are you proposing to use say Optim.jl to solve this univariate convex minimization problem?

Since it's stricktly unimodal (since it's convex), we can implement a [golden-section search](https://en.wikipedia.org/wiki/Golden-section_search) which is implemented in a few lines. A TODO is ok

Reply:
The dual for the RelativeEntropyCone I gave was incorrect. When changing it, we get a closed form expression for `b` so no need to solve an univariate optimization problem anymore, see https://github.com/JuliaOpt/MathOptInterface.jl/pull/1028